internal/impl/elasticsearch/output.go

```diff
@@ -98,20 +98,23 @@ func esoConfigFromParsed(pConf *service.ParsedConfig) (conf esoConfig, err error
 		return
 	}
 
-	httpClient := &http.Client{
-		Timeout: timeout,
-	}
+	transport := &http.Transport{}
 
 	var tlsConf *tls.Config
 	var tlsEnabled bool
 	if tlsConf, tlsEnabled, err = pConf.FieldTLSToggled(esoFieldTLS); err != nil {
 		return
 	} else if tlsEnabled {
-		httpClient.Transport = &http.Transport{
-			TLSClientConfig: tlsConf,
-		}
+		transport.TLSClientConfig = tlsConf
 	}
 
+	httpClient := &http.Client{
+		Transport: transport,
+		Timeout:   timeout,
+	}
+
+	conf.clientOpts = append(conf.clientOpts, elastic.SetHttpClient(httpClient))
```

Comment:
Is setting the HTTP client not the only change that's required here?

Reply:
```suggestion
	if !pConf.Contains(esoFieldAWS) {
		conf.clientOpts = append(conf.clientOpts, elastic.SetHttpClient(httpClient))
	}
```

should we do this instead? @gregfurman

Reply:
pretty sure there is an issue with the .Contains method - returning true even if you don't set an AWS field 😥